pom.xml

```diff
@@ -1552,6 +1553,11 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>org.apache.zookeeper</groupId>
+        <artifactId>zookeeper</artifactId>
```

Comment:
Do we need update of `curator-framework` too? Since the Curator Framework is a high-level API that simplifies using ZooKeeper.
You can try 2.12.0 version of `curator-framework`.

Reply:
Please see my comment on https://issues.apache.org/jira/browse/DRILL-6540.

Reply:
I've tried upgrading all curator dependencies to 2.12.0, it seems to work as expected but curator-client's size is much larger than current one's, so one needs to update max size of drill-jdbc-all module by extra 1-2 MB. It is better to be handled as another task.
However I did add curator dependencies to dependencyManagement explicitly.

Reply:
You have specified `Zookeeper` and `Curator` dependencies in dependencyManagement block and it's fine. It is good for controlling the versions of these libraries and their exclusions over the whole project.
But still this dependencies explicitly are used in the Drill project modules as transitive dependencies. Could you specify them explicitly in dependencies for Drill modules to avoid of using them as transitive dependencies?